config.example.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+return [
+    "database" => [
+        "dsn" => "mysql:host=;dbname=",
+        "username" => "prabeg",
```

Comment:
retrieve this value from `.env` file because configuration can defer from user to user